wouso/games/teamquest/models.py

```diff
@@ -13,10 +16,32 @@ def is_head(self):
         return self.group.head == self
 
 
+class TeamQuestLevel(models.Model):
+    questions = models.ManyToManyField(Question)
+    quest = models.ForeignKey('TeamQuest', null=True, blank=True, related_name='levels')
+    bonus = models.IntegerField(default=0)
+
+    @classmethod
+    def create(cls, quest, bonus, questions):
+        new_level = cls.objects.create(quest=quest, bonus=bonus)
+        for q in questions:
+            new_level.questions.add(q)
+        return new_level
+
+    def add_question(self, question):
+        self.questions.add(question)
+
+    def remove_question(self, question):
+        if question in self.questions.all():
+            self.questions.remove(question)
+
+    def set_quest(self, quest):
```

Comment:
I would remove the method, as we would just use `level.quest = quest` in the implementation.